util/execdetails/execdetails.go

```diff
@@ -52,3 +54,55 @@ func (d ExecDetails) String() string {
 	}
 	return strings.Join(parts, " ")
 }
+
+// ExecStats collects executors's execution info.
+type ExecStats map[string]*ExecStat
```

Comment:
Why use a map to store the runtime statistics of all the executors? Maybe its simpler to make each executor to hold a `ExecStat` object?

Reply:
Because I want to get  `RuntimeStat` from a `PhysicalPlan`, so this map will give `ExplainID to RuntimeStat` mapping. or do we have better way to get `Executor` from an `PhysicalPlan`~? :cat: 

And it seem some physical is "1 to N" relationship with executor. (e.g. IndexLookUpExecutor)

Except that map lookup also easy to combine coprocessor's RuntimeStats in furture~?